core/src/test/java/com/datastax/oss/driver/internal/core/config/typesafe/DefaultDriverConfigLoaderTest.java

```diff
@@ -128,11 +129,11 @@ public void should_reload_if_forced() {
 
     configSource.set("int1 = 43");
 
-    eventBus.fire(ForceReloadConfigEvent.INSTANCE);
+    CompletionStage<Void> reloaded = loader.reload();
     adminExecutor.waitForNonScheduledTasks();
 
     assertThat(initialConfig).hasIntOption(MockOptions.INT1, 43);
-    Mockito.verify(eventBus).fire(ConfigChangeEvent.INSTANCE);
```

Comment:
`reload()` is still `fire()` that event, so why we removed that check?

Reply:
Right, I was thinking of the other event that I removed, this one is indeed still fired.
Will revert 👍